src/main/basecamp.ts

```diff
@@ -68,6 +68,18 @@ export function sendBasecampPutRequest(requestUrl: string, requestPayload: JsonO
     return JSON.parse(response.getContentText());
 }
 
+export function sendBasecampDeleteRequest(requestUrl: string): void {
```

Comment:
technically this isn't a basecamp delete then, this is still using a basecamp put. the wrapper function outside of this should then be appropriately named as delete todo (which it is)

Reply:
so basically i think it's ok to not have this function, and delete todo should just directly use basecamp delete

Reply:
Got it

Reply:
It's deleted